components/pedigree/resources/src/main/resources/pedigree/view/person.js

```diff
@@ -1289,7 +1364,12 @@ define([
                 causal_genes:    {value : this.getCausalGenes(), disabled: false},
                 carrier_genes:   {value : this.getCarrierGenes(), disabled: false},
                 rejected_genes:  {value : rejectedGeneList, disabled: true, inactive: (rejectedGeneList.length == 0)},
-                rejected_candidate_genes:  {value : rejectedCandidateGeneList, disabled: true, inactive: (rejectedCandidateGeneList.length == 0)},
+                rejectedcandidate_genes:  {value : rejectedCandidateGeneList, disabled: true, inactive: (rejectedCandidateGeneList.length == 0)},
```

Comment:
Anything related to genes should be populated by iterating through supported statuses instead of hardcoding them.